### ISSUE TITLE
darkflow training stop

### ISSUE THREAD

**Issue by Muntasir-mn** [thtrieu/darkflow#975]
When should I stop training? Now my avg loss is  1.0829794632627456e-06 @thtrieu

**Comment by Alemolet** [comment]
As far as I know, a lot of people have empirically stated that a loss <0.1 should be a good place to stop, in order to avoid overfitting. However, low loss doesn't mean good accuracy and quality. You should train for 40.000 iterations ~ even though the loss seems stuck. I myself noticed that this is actually true: apart from my loss being stucked at 8.0, the weights obtained at 10.000 give worse results than the ones obtained at 35.000. If you are wondering, I don't know exactly why this magic happens, people on GitHub and Stackoverflow noticed it.
